lib/node_modules/@stdlib/array/base/take-map/README.md

```diff
@@ -0,0 +1,117 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# take-map
+
+> Take elements from an array, applies a mapping function using a callback, and returns a new array.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var takeMap = require('@stdlib/array/base/take-map');
+```
+
+### takeMap( x, indices, mode, clbk)
+
+Combines the take and map operations into a single API call.
+
+```javascript
+var takeMap = require('@stdlib/array/base/take-map');
+var x = [ 1, 2, 3, 4 ];
+
+function customMapping(value) {
+    return value * 2;
+}
+
+var y = takeMap( x, [ 1, 3 ], 'throw', customMapping );
+// returns [ 4, 8 ]
+```
+
+The function supports the following parameters:
+
+-   **x**: input array.
+-   **indices**: list of indices.
+-   **mode**: index [mode][@stdlib/ndarray/base/ind].
+-   **clbk**: function to apply.
+
+</section>
+
+<!-- /.usage -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var takeMap = require('@stdlib/array/base/take-map');
```

Comment:
It is our convention that the examples section in the `README.md` should match the examples in the `examples/index.js` file.